java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -112,4 +112,40 @@ public interface GenericClusterCommands {
      * }</pre>
      */
     CompletableFuture<Object[]> exec(ClusterTransaction transaction, SingleNodeRoute route);
+
+    /**
+     * Returns a random key. <br>
```

Comment:
Why don't we mentioned database in cluster mode? 
```suggestion
     * Returns a random key from the currently selected database. <br>
```

Reply:
currently selected database is used for standalone (select command)

we can write about the routing that determents where are is the randomkey selected from